### DISCUSSION TITLE
Add VSCode GitHub Copilot hotkey to only accept suggestion but up to current line

### DISCUSSION THREAD

**Discussion by nikivdev** [community/community#19510] [Copilot Conversations]
Sometimes Copilot suggests something correct but only up to a certain line. So like example here:

![FWM_sDjWAAAx09Y](https://user-images.githubusercontent.com/6391776/175831401-d8e97d50-94c9-42b1-b757-15f16f4eda36.png)

I want to only accept the two lines:

```
                      setActiveStep(t("Select"))
                      setReturnData({})
```

The rest is wrong. But I can't really do this. I have to type out the two lines myself or accept the whole block which is wrong and then delete line by line.

I'd love a dedicated hotkey to just accept suggestion up to the current line. Then I can go on next line and then press it again to accept the second line.

Would solve this use case perfectly for me.

**Comment by itsgareth** [comment]
yeah this would be a nice feature to have possibly something along the lines of `tab + number_of_lines` in this case `tab + 2` will just accept the first two lines.

its unfortunate that this isn't a feature yet.

**Reply by nikivdev** [reply]
I love the `tab + <number>` suggestion. Hope GitHub team looks into it.

**Reply by nikivdev** [reply]
Although `tab` is not a modifier key I believe. I don't think VSCode can trigger that as hotkey. 🤔
